Source/cubfan135/style.css

```diff
@@ -341,3 +341,17 @@ body.dark .area::placeholder {
   filter: sepia(180%);
   outline: none;
 }
+
+@media screen and (prefers-reduced-motion: reduce) {
+  .area:active,
+  .area:hover{
+    transform: none !important;
+  }
+
+  /* style for the .area class elements except the download button */
+  .area:active:not(#downloader),
```

Comment:
On `:active` state all elements get a sepia filter (from line 341), so this line is not needed. It won't do much but I just pointed out.

Reply:
This is a very small change, so I'll change it myself and accept the PR.

Reply:
You are right, there is a color transition on the `:focus` state which makes having the style on the `:active` state redundant. Thanks for pointing that out.